packages/components/src/components/shell-panel/shell-panel.tsx

```diff
@@ -128,7 +128,7 @@ export class ShellPanel extends LitElement {
   /**
    * When `layout` is `vertical`, specifies the width of the component.
    *
-   * @deprecated Use the `width` property instead.
+   * @deprecated in v3.0, removal target v6.0 -  Use the `width` property instead.
```

Comment:
Consistency suggestion:
```suggestion
   * @deprecated in v3.0.0, removal target v6.0.0 -  Use the `width` property instead.
```